tests/gql_behave/tests/memgraph_V1/features/memgraph_exists.feature

```diff
@@ -65,3 +80,163 @@ Feature: WHERE exists
           MATCH (n:One) WHERE exists((n)<-[:TYPE]-()) RETURN n.prop;
           """
       Then the result should be empty
+
+  Scenario: Test exists with destination node label
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[]->(:Two)) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
+  Scenario: Test exists with wrong destination node label
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two)
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[]->(:Three)) RETURN n.prop;
+          """
+      Then the result should be empty
+
+  Scenario: Test exists with destination node property
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two {prop: 2})
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[]->({prop: 2})) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
+  Scenario: Test exists with wrong destination node property
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE]->(:Two {prop: 2})
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[]->({prop: 3})) RETURN n.prop;
+          """
+      Then the result should be empty
+
+  Scenario: Test exists with edge property
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE {prop: 1}]->(:Two {prop: 2})
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[:TYPE {prop: 1}]->()) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
+  Scenario: Test exists with wrong edge property
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE {prop: 1}]->(:Two {prop: 2})
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[:TYPE {prop: 2}]->()) RETURN n.prop;
+          """
+      Then the result should be empty
+
+  Scenario: Test exists with both edge property and node label property
+      Given an empty graph
+      And having executed:
+          """
+          CREATE (:One {prop:1})-[:TYPE {prop: 1}]->(:Two {prop: 2})
+          """
+      When executing query:
+          """
+          MATCH (n:One) WHERE exists((n)-[:TYPE {prop: 1}]->(:Two {prop: 2})) RETURN n.prop;
+          """
+      Then the result should be:
+          | n.prop |
+          | 1      |
+
```

Comment:
Would it make sense to add these tests?
* Test exists with wrong edge property and correct node label property
* Test exists with correct edge property and wrong node label property
* Test exists with wrong edge property and node label property

Reply:
will add